dev/com.ibm.ws.jaxws.ejb/bnd.bnd

```diff
@@ -52,47 +49,34 @@ Service-Component: \
      properties:="service.vendor=IBM"
 
 Export-Package: \
-    com.ibm.ws.jaxws.ejb
+   com.ibm.ws.jaxws.ejb
 
-# Using version=! in order to not have a version attached to the import for packages that were removed
-# from Java after Java 8.  Doing this keeps the import like before Java 11 support. It will get the 
-# packages from Java when using Java 8 or earlier and from the new shipped bundles for Java 9 and later.
 Import-Package: \
-   !com.ibm.ws.jaxws.ejb, \
-   javax.xml.ws.*;version="[2.2,3)", \
-   org.apache.cxf.*;version="[2.6.2,3)", \
-   javax.xml.soap;version=!,\
-   com.ibm.ws.jaxws;version="[2.0,2.4)", \
-   com.ibm.ws.jaxws.*;version="[2.0,2.4)", \
-   *
+  !com.ibm.ws.jaxws.ejb, \
+  javax.xml.ws.*;version="[2.2,3)", \
+  org.apache.cxf.*;version="[3.2,4.0)", \
+  *
 
 Private-Package:\
-   com.ibm.ws.jaxws.ejb.*
-
-instrument.classesExcludes: com/ibm/ws/jaxws/ejb/internal/resources/*.class
```

Comment:
This needs to stay in.  The only reason it wasn't there in the 2.3 one is because we took the class files as is and they weren't instrumented when we took them.